New package: FastHartleyTransform v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: FastHartleyTransform
- Repository: https://github.com/EHTJulia/FastHartleyTransform.jl
- Created by: @kazuakiyama
- Version: v0.1.0
- Commit: f00180562b069abb7b5f4bd003d33c53af8df2ab
- Git reference: v0.1.0
- Description: A Julia implementation of the Fast Hartley Transform (FHT)
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
The first release. It provides implementations of all interface functions defined in AbstractFastHartleyTransform.jl, for CPUs (based on FFTW.jl) and NVIDIA GPUs (based on CUDA.jl's CUFFT).
`````
<!-- END RELEASE NOTES -->

kazuakiyama commented:
I would block merging this version as I made another fix (Compat Helper's pull request), and release the version again.

JuliaHubRegistrator committed:
New package: FastHartleyTransform v0.1.0

UUID: e0605d58-1a4b-427b-8ab9-d16fed775e83
Repo: https://github.com/EHTJulia/FastHartleyTransform.jl.git
Tree: 9a51d17e2855dda2ae13bcbe8de06b1acb39922d

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
